src/index.js

```diff
@@ -245,7 +245,8 @@ function createLoadableComponent(loadFn, options) {
           pastDelay: this.state.pastDelay,
           timedOut: this.state.timedOut,
           error: this.state.error,
-          retry: this.retry
+          retry: this.retry,
+          initialProps: this.props
```

Comment:
Why to name it as `initialProps`? I would name it as `props` for it will change when props changes.

Reply:
`props.props` looks something strange. I think you are right  that `initialProps` is not so clear.

Reply:
I have changed it now.